templates/Admin/dashboard.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends "Admin/base.html.twig" %}
+{% block body %}
+    <div>
+        <li>
+            <ul style="list-style-type: none">
+                <li style="float: left; padding: 10px"><a href="#">My account</a></li>
+                <li style="float: left; padding: 10px"><a href="#">Users</a></li>
+                <li style="float: left; padding: 10px"><a href="#">Programmes</a></li>
+                <li style="float: left; padding: 10px"><a href="{{ path('admin_logout') }}">Logout</a></li>
```

Comment:
butonul de logout poti sa il pui in base template, ca ai vrea de pe orice pagina sa poti sa dai logout, nu doar de pe homepage, dar ar trebui afisat doar cand cineva e logat, ca ar fi ciudat sa intri pe un site si sa ai un buton de logout fara ca tu sa te fi logat.
O bucatica de cod care te poate ajuta cu asta:
{% if is_granted('ROLE_USER') %}
Hi {{ app.user.username }}
{% endif %}